pkg/network/node/iptables.go

```diff
@@ -189,16 +183,14 @@ func (n *NodeIPTables) getNodeIPTablesChains() []Chain {
 			rules:    filterRules,
 		},
 	)
-	if !n.masqueradeServices {
-		masq2Rules = append(masq2Rules, []string{"-j", "MASQUERADE"})
-		chainArray = append(chainArray,
-			Chain{
-				table: "nat",
-				name:  "OPENSHIFT-MASQUERADE-2",
-				rules: masq2Rules,
-			},
-		)
-	}
+	masq2Rules = append(masq2Rules, []string{"-j", "MASQUERADE"})
+	chainArray = append(chainArray,
+		Chain{
+			table: "nat",
+			name:  "OPENSHIFT-MASQUERADE-2",
+			rules: masq2Rules,
+		},
+	)
```

Comment:
...and then this can be part of the previous `chainArray = append(chainArray,` (in the order `OPENSHIFT-MASQUERADE`, `OPENSHIFT-MASQUERADE-2`, `OPENSHIFT-FIREWALL-FORWARD`)

Reply:
Since I'm here, do I join the next append ChainArray too?
```
	// HACK: block access to MCS until we can secure it properly. Note that we share
	// the same chain between OUTPUT and FORWARD.
	chainArray = append(chainArray,
		Chain{
			table:    "filter",
			name:     "OPENSHIFT-BLOCK-OUTPUT",
			srcChain: "OUTPUT",
			srcRule:  []string{"-m", "comment", "--comment", "firewall overrides"},
			rules: [][]string{
				{"-p", "tcp", "-m", "tcp", "--dport", "22623", "-j", "REJECT"},
				{"-p", "tcp", "-m", "tcp", "--dport", "22624", "-j", "REJECT"},
			},
		},
		Chain{
			table:    "filter",
			name:     "OPENSHIFT-BLOCK-OUTPUT",
			srcChain: "FORWARD",
			srcRule:  []string{"-m", "comment", "--comment", "firewall overrides"},
			rules:    nil,
		},
	)
```

Reply:
The masquerading rules and MCS-blocking rules are unrelated, so it makes sense to have them be separate. As opposed to the masquerade stuff, where it was previously in two pieces just because we needed to support doing it two different ways.